Makefile

```diff
@@ -1,5 +1,5 @@
 .PHONY: default
-default: start
```

Comment:
You ok with this?

Reply:
100%. There's a `make` way of setting the default target too (but I forgot about it when I wrote the makefile..) Purely FYI (and it's make 3.81 and above only):
```
.DEFAULT_GOAL := install
```

etc :)